src/redux/books/categories/categories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Categories = () => (
+  <section className="categories">
+    <h2>Work in  progress</h2>
+  </section>
+);
+
+export default Categories;
```

Comment:
- [ ] I would recommend following a good project structure. It should follow the `ducks` pattern. Keep the `categories` folder outside the `books` folder.